### ISSUE TITLE
Interlock -- Core-Command AdminState / OperatingState Endpoint Changes

### ISSUE THREAD

**Issue by tsconn23** [edgexfoundry/blackbox-testing#200]
Please see the source issue in the edgex-go repo https://github.com/edgexfoundry/edgex-go/issues/1244 for context.

Core-command blackbox tests for these API endpoints will need to be refactored to support the new request format
- /device/{id}/adminstate/{adminState}
- /device/{id}/opstate/{opState}
- /device/name/{name}/adminstate/{adminState}
- /device/name/{name}/opstate/{opState}

NOTE: Work hasn't yet begun on the above issue in edgex-go.

**Comment by tsconn23** [comment]
FYI -- I had to go ahead and approve the PR https://github.com/edgexfoundry/edgex-go/pull/1278 due to some merge timing/sequencing. That means there will be a few tests related to updating Admin/Operating State on devices until changes for this issue are completed. @AnthonyMBonafide @cloudxxx8

**Comment by cloudxxx8** [comment]
add @cherrycl for awareness

**Comment by AnthonyMBonafide** [comment]
I have changes ready to go. I am just waiting on changes in edgex-go for issue #1285(https://github.com/edgexfoundry/edgex-go/issues/1285) since that fixes issues which will cause these updated tests to incorrectly fail.

**Comment by cloudxxx8** [comment]
fixed by #206

**Comment by cloudxxx8** [comment]
reopen this until edgexfoundry/edgex-go#1296 resolved, and we will create another PR to fix it

**Comment by tsconn23** [comment]
Closing this issue b/c we decided to remove the endpoints from core-command after all. Thus the following PR actually removes these tests altogether.
https://github.com/edgexfoundry/blackbox-testing/pull/227
